core/client/fs/src/main/java/alluxio/client/file/cache/CacheManager.java

```diff
@@ -106,6 +107,13 @@ public static CacheManager get(AlluxioConfiguration conf) throws IOException {
      */
     static CacheManager create(AlluxioConfiguration conf) throws IOException {
       try {
+        boolean isShadowCacheEnabled =
+            conf.getBoolean(PropertyKey.USER_CLIENT_CACHE_SHADOW_ENABLED);
+        if (isShadowCacheEnabled) {
+          int shadowCacheWindow = conf.getInt(PropertyKey.USER_CLIENT_CACHE_SHADOW_WINDOW);
```

Comment:
Use `getMs`, in this way , users can set this property using strings like `24h` or  `5min`
```suggestion
          long shadowCacheWindowMs = conf.getMs(PropertyKey.USER_CLIENT_CACHE_SHADOW_WINDOW);
```
if we are expecting sec as arg, you need to convert from ms (what returned by getMs to sec)